### PR TITLE
Efi setup for removable devices

### PR DESCRIPTION
**Pull request by schaefi**
Added target_removable attribute
    
Indicate if the target disk for oem images is deployed to a removable device e.g a USB stick or not. This only affects the EFI setup if requested and in the end avoids the creation of a custom boot menu entry in the firmware of the target machine on first boot. This is related to bnc#993130

### CODE REVIEW

**Comment by schaefi** [comment]
Autogenerated code:

* kiwi/schema/kiwi.rng 
* kiwi/xml_parse.py

please do not review those two, thanks

File: kiwi/schema/kiwi.rnc [hunk 1]
```diff
@@ -1608,6 +1608,14 @@ div {
...
         ## desired target by calling: blockdev --report device
         attribute target_blocksize { xsd:nonNegativeInteger }
+    k.type.target_removable =
+        ## Indicate if the target disk for oem images is deployed
+        ## to a removable device e.g a USB stick or not. This only
+        ## affects the EFI setup if requested and in the end avoids
+        ## the creation of a custom boot menu entry in the firmware
+        ## of the target machine. By default the target disk is
+        ## expected to be solid
+        attribute target_removable { xsd:boolean }
```
Comment (schaefi) [inline kiwi/schema/kiwi.rnc]: makes sense, I updated it

**Comment by schaefi** [comment]
@davidcassany guess we are good to go ?

**Comment by davidcassany** [comment]
@schaefi Yes I think we are good :smiley:

**Pull request merged**